.vsts-pipelines/builds/ci-public.yml

```diff
@@ -0,0 +1,16 @@
+trigger:
+- master
+- release/*
+
+resources:
+  repositories:
+  - repository: buildtools
+    type: github
+    endpoint: DotNet-Bot GitHub Connection
+    name: aspnet/BuildTools
+    ref: refs/heads/release/2.2
+
+phases:
+  - template: .vsts-pipelines/templates/phases/default-build.yml@buildtools
```

Comment:
I don't know where this comes from. Is it part of our internal SDK, or built into VSTS?

Reply:
It's part of the internal SDK, the `@buildtools` references the `repositories` section above where we link in https://github.com/aspnet/BuildTools . All our other repos use this same template.

Reply:
We're not actually using this part yet anyway due to a problem in VSTS.